### PR TITLE
wrap the title of a panel and dashboard search results. fixes #7203

### LINKED ISSUE #7203: [Bug] Dashboard list does not support long names or many tags
Please include this information:
- What Grafana version are you using?
v4.0.2
- What datasource are you using?
Influxdb
- What OS are you running grafana on?
Debian Jessie
- What did you do?
- What was the expected result?
The text or tags to wrap
- What happened instead?
The text overlays over the tags

The dashboard list needs to wrap the text and/or move the tags below the dashboard name.

dashboard list

### PR DESCRIPTION
**Pull request by xginn8**
2017-01-10-221325_1920x1080_scrot

### CODE REVIEW

File: public/sass/pages/_dashboard.scss [hunk 1]
```diff
@@ -57,6 +57,7 @@ div.flot-text {
   min-height: 9px;
   padding-top: 4px;
   cursor: pointer;
+  word-wrap: break-word;
```
Comment (torkelo) [inline public/sass/pages/_dashboard.scss]: this does not look right. The dashboard title should never wrap (the header cannot be higher or things break).

**Comment by torkelo** [comment]
I would prefer a max width and ellipsis (so full title is not shown)

**Comment by torkelo** [comment]
@mattttt thoughts?

**Comment by magnuspwhite** [comment]
@torkelo The example I used in the ticket (#7203) shows with large infrastructure or a requirement for high granularity it is easy to exceed the character limit before the name and tags start to overlap.

Using an ellipsis to hide the text is as good as having a fixed character limit because you loose the detail required.

This may be our specific implementation with 7 organisations, 30+ dashboards per and 

Alternatives may include:
- maybe to look into being able to group dashboards under titles
- have the ellipsis by default and an option to enable unlimited length names

However, the issue still applies if you have a 1 word dashboard name but many tags.

**Comment by torkelo** [comment]
Understand that but do no think allowing the text to wrap here is the right way forward. 

I think dashboard folders (https://github.com/grafana/grafana/issues/6475) and improvements to search (widinging it), https://github.com/grafana/grafana/issues/6475 will help.

**Comment by magnuspwhite** [comment]
I think folders will greatly improve the organisation on dashboard and will overcome this issue.

**Comment by torkelo** [comment]
merging this as it's such a small change, and even though i prefer to enforce no wrap here and use ellipsis, I think allowing wrap is ok until I redo the search dropdown (make it wider), and  until we introduce dashboard folders

**Pull request merged**